src/Components/CreateImageWizardV2/steps/Review/Footer.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { useState, useEffect } from 'react';
+
+import {
+  Button,
+  Dropdown,
+  DropdownItem,
+  DropdownList,
+  MenuToggle,
+  MenuToggleElement,
+  WizardFooterWrapper,
+  useWizardContext,
+} from '@patternfly/react-core';
+import { SpinnerIcon } from '@patternfly/react-icons';
+import { useChrome } from '@redhat-cloud-services/frontend-components/useChrome';
+import { useNavigate, useParams } from 'react-router-dom';
+
+import {
+  useComposeBlueprintMutation,
+  useCreateBlueprintMutation,
+} from '../../../../store/imageBuilderApi';
+import { resolveRelPath } from '../../../../Utilities/path';
+import { mapRequestFromState } from '../../utilities/requestMapper';
+
+const ReviewWizardFooter = () => {
+  const { goToPrevStep, close } = useWizardContext();
+  const [
+    createBlueprint,
+    { isLoading: isCreationLoading, isSuccess: isCreationSuccess },
+  ] = useCreateBlueprintMutation();
+  const [buildBlueprint, { isLoading: isBuildLoading }] =
+    useComposeBlueprintMutation();
+  const { auth } = useChrome();
+  const navigate = useNavigate();
+  const { composeId } = useParams();
+  const [isOpen, setIsOpen] = useState(false);
+
+  const onToggleClick = () => {
+    setIsOpen(!isOpen);
+  };
+
+  useEffect(() => {
+    if (isCreationSuccess) {
+      navigate(resolveRelPath(''));
+    }
+  }, [isCreationSuccess, navigate]);
+
+  const getBlueprintPayload = async () => {
+    const userData = await auth?.getUser();
+    const orgId = userData?.identity?.internal?.org_id;
+    const requestBody = orgId && mapRequestFromState(orgId);
+    return requestBody;
+  };
+
+  const onSave = async () => {
+    const requestBody = await getBlueprintPayload();
+    setIsOpen(false);
+    !composeId &&
+      requestBody &&
+      createBlueprint({ createBlueprintRequest: requestBody });
+  };
+
+  const onSaveAndBuild = async () => {
+    const requestBody = await getBlueprintPayload();
+    setIsOpen(false);
+    const blueprint =
+      !composeId &&
+      requestBody &&
+      (await createBlueprint({ createBlueprintRequest: requestBody }).unwrap());
+    blueprint && buildBlueprint({ id: blueprint.id });
+    setIsOpen(false);
```

Comment:
This is also being set to false a few lines above, one of them is probably a duplicate.